### ISSUE TITLE
Jest installation and use of basic example is unclear / throws Error.

### ISSUE THREAD

**Issue by mandysimon88** [blainekasten/enzyme-matchers#58]
Using jest-enzyme, here is my test file, copied directly from your examples:

```
import React from 'react'
import {mount, shallow} from 'enzyme'

function Fixture() {
  return (
    <div>
      <input id="checked" defaultChecked />
      <input id="not" defaultChecked={false} />
    </div>
  );
}

describe('wrapper', () => {
  it('works', () => {
    const wrapper = mount(<Fixture />); // mount/render/shallow when applicable
    expect(wrapper.find('#checked')).toBeChecked();
    expect(wrapper.find('#not')).not.toBeChecked();
  })
})
```

I have installed as devDependencies jest-enzyme, enzyme, react-addons-test-utils.

I get this failure message:
```
> CI=true react-scripts test --env=jsdom

 FAIL  src/__tests__/ListModalSpec.js
  ● wrapper › works

    TypeError: expect(...).toBeChecked is not a function

      at Object.<anonymous> (src/__tests__/ListModalSpec.js:16:38)
      at process._tickCallback (internal/process/next_tick.js:103:7)
```

How am I holding it wrong?

**Comment by blainekasten** [comment]
Did you follow the setup for jest-enzyme? https://github.com/blainekasten/enzyme-matchers/blob/master/packages/jest-enzyme/README.md#setup

**Comment by mandysimon88** [comment]
Yes. Here is my package.json:
```
{
  "name": "react-example-project",
  "version": "1.0.0",
  "description": "Simple React project",
  "main": "index.js",
  "repository": {},
  "scripts": {
    "test": "CI=true react-scripts test --env=jsdom",
    "watch_test": "react-scripts test --env=jsdom"
  },
  "author": "Amanda Simon",
  "license": "UNLICENSED",
  "dependencies": {
    "babel-cli": "^6.18.0",
    "babel-plugin-transform-react-jsx": "^6.8.0",
    "babel-preset-es2015": "^6.18.0",
    "babel-plugin-add-module-exports": "^0.2.1",
    "react": "^15.4.1",
    "react-dom": "^15.4.1",
    "react-router": "^3.0.0"
  },
  "devDependencies": {
    "enzyme": "^2.7.1",
    "jest-enzyme": "^2.1.0",
    "react-addons-test-utils": "^15.4.2",
    "react-scripts": "0.8.4"
  },
  "babel": {
    "presets": [
      "es2015"
    ],
    "plugins": [
      "transform-react-jsx",
      "add-module-exports"
    ]
  },
  "jest": {
    "setupTestFrameworkScriptFile": "./node_modules/jest-enzyme/lib/index.js"
  },
}
```

**Comment by mandysimon88** [comment]
`react-scripts` currently uses `jest@17.0.2`

**Comment by blainekasten** [comment]
Ah, so react-scripts handles jest config a little bit differently. This PR show you how to setup jest-enzyme with react-scripts https://github.com/facebookincubator/create-react-app/pull/994/files

**Comment by mandysimon88** [comment]
Thanks! You might add that link & explanation to the jest-enzyme README. 

I'd started to dig through node_modules figuring `react-scripts` might be the problem but hadn't put it together yet that I could pass it a setup file. Thank you!

**Comment by mandysimon88** [comment]
Okay, I got the first example above working by following the instructions (making a setupTests file that imports `jest-enzyme`). I get a "Pass". But, other matcher examples still fail. For example:
```
import React from 'react'
import {mount, shallow} from 'enzyme'

function Fixture() {
  return (
    <div>
      <span className="foo" />
      <span className="bar baz" />
    </div>
  );
}

describe('wrapper', () => {
  it('works', () => {
    const wrapper = mount(<Fixture />); // mount/render/shallow when applicable
    expect(wrapper.find('span')).toBePresent();
    expect(wrapper.find('ul')).not.toBePresent();
  })
})
```
Produces:
```
 FAIL  src/__tests__/ListModalSpec.js
  ● wrapper › works

    TypeError: Cannot read property '_tag' of undefined

      at Object.<anonymous> (src/__tests__/ListModalSpec.js:17:34)
      at process._tickCallback (internal/process/next_tick.js:103:7)
```

Sorry I'm having such a hard time figuring this out!

**Comment by mandysimon88** [comment]
This seems to be same error as: https://github.com/blainekasten/enzyme-matchers/issues/55 since the other matcher examples work.

**Comment by blainekasten** [comment]
What version of React are you using?

**Comment by blainekasten** [comment]
I just copied your test into my repo and it passed just fine. It must be a react version I wasn't planning for..

**Comment by blainekasten** [comment]
> Thanks! You might add that link & explanation to the jest-enzyme README.

> I'd started to dig through node_modules figuring react-scripts might be the problem but hadn't put it together yet that I could pass it a setup file. Thank you!

A smart idea. Want to submit a PR for it?

**Comment by blainekasten** [comment]
> Sorry I'm having such a hard time figuring this out! 

No problem at all! Glad you are finding this to be helpful!

**Comment by mandysimon88** [comment]
15.4.1
```
"dependencies": {
    "babel-cli": "^6.18.0",
    "babel-plugin-transform-react-jsx": "^6.8.0",
    "babel-preset-es2015": "^6.18.0",
    "babel-plugin-add-module-exports": "^0.2.1",
    "react": "^15.4.1",
    "react-dom": "^15.4.1",
    "react-router": "^3.0.0"
  },
  "devDependencies": {
    "enzyme": "^2.7.1",
    "jest-enzyme": "^2.1.0",
    "react-addons-test-utils": "^15.4.2",
    "react-scripts": "0.8.4"
  },
```

**Comment by blainekasten** [comment]
Yay, I reproduced it on 15.4.2! I'll try and find a fix. Thanks so much! I'm not sure if there is a work around right now

**Comment by mandysimon88** [comment]
Awesome! Thank you!

**Comment by blainekasten** [comment]
Can you test `2.1.1-rc1` please

**Comment by blainekasten** [comment]
I published `2.1.1` which seemed to fix this. Thanks for the reports!
